apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -31,15 +41,18 @@ class SOMATileDBContext:
 
     tiledb_ctx: tiledb.Ctx = _build_default_tiledb_ctx()
 
-    timestamp: Optional[int] = attrs.field(default=None)
+    timestamp_ms: Optional[int] = attrs.field(
+        default=None, converter=_maybe_timestamp_ms
+    )
     """
-    Timestamp for operations on SOMA objects, in milliseconds since Unix epoch.
+    Default timestamp for operations on SOMA objects, in Unix millis.
```

Comment:
Not sure what "Unix millis" indicates.  If actually `milliseconds since the Python (aka Unix) epoch`, it may be clearer to fully state that. That would match the Python `time.time()` documentation.